### PR TITLE
 Add PVC and PV some types

### PR DESCRIPTION
**Pull request by riverzhang**
Add PVC some type include stauts capacity accessmodes storageclass
 Fix list PVC's Volume err
 Add PV some type include accessmodes and storageclass

### CODE REVIEW

**Comment by riverzhang** [comment]
screenshot from 2017-08-11 14-31-22
screenshot from 2017-08-11 14-30-29

**Comment by riverzhang** [comment]
@floreks @maciaszczykm

**Comment by riverzhang** [comment]
example PV:
screenshot from 2017-08-14 13-41-34

example PVC:
screenshot from 2017-08-14 13-42-07

File: src/app/frontend/persistentvolume/list/cardlist.html [hunk 1]
```diff
@@ -40,15 +40,15 @@
...
     </kd-resource-card-header-column>
     <kd-resource-card-header-column size="small"
                                     grow="2">
-      [[Capacity|Persistent volume list header: capacity.]]
+      [[AccessModes|Persistent volume list header: access modes.]]
     </kd-resource-card-header-column>
     <kd-resource-card-header-column size="small"
                                     grow="2">
-      [[Access modes|Persistent volume list header: access modes.]]
+      [[ReclaimPolicy|Persistent volume list header: reclaim policy.]]
```
Comment (maciaszczykm) [inline src/app/frontend/persistentvolume/list/cardlist.html]: Still needs to be updated. You have changed only descriptions.

File: src/app/frontend/persistentvolumeclaim/list/cardlist.html [hunk 2]
```diff
@@ -45,11 +45,23 @@
...
-      [[Labels|Label 'Labels' which appears as a column label in the table of persistent volume claim (persistent volume claim list view).]]
+      [[Capacity|Label 'Capacity' which appears as a column label in the table of persistent volume claims (persistent volume claim list view).]]
+    </kd-resource-card-header-column>
+    <kd-resource-card-header-column size="small"
+                                    grow="2">
+      [[AccessModes|Label 'AccessModes' which appears as a column label in the table of persistent volume claims (persistent volume claim list view).]]
+    </kd-resource-card-header-column>
+    <kd-resource-card-header-column size="small"
+                                    grow="2">
+      [[StorageClass|Label 'StorageClass' which appears as a column label in the table of persistent volume claims (persistent volume claim list view).]]
```
Comment (maciaszczykm) [inline src/app/frontend/persistentvolumeclaim/list/cardlist.html]: Wrong values here, as I said before.

File: src/app/frontend/persistentvolume/list/cardlist.html (hunk 1 above)
Comment (riverzhang) [inline src/app/frontend/persistentvolume/list/cardlist.html]: ok，  "translated is missing"     I do not understand what it means

**Pull request merged**